Makefile

```diff
@@ -135,8 +135,12 @@ php7-image: ## Build the PHP7 test runtime container image.
 python-image: ## Build the Python test runtime container image.
 	docker build -t ${IMAGE_PREFIX}python:${TEST_INFRA_VERSION} containers/runtime/python
 
-ready-image: ## Build the ready init container image.
-	docker build -t ${INIT_IMAGE_PREFIX}ready:${TEST_INFRA_VERSION} -f containers/init/ready/Dockerfile .
+# Build the driver container image at the $DRIVER_VERSION
```

Comment:
I am not sure if we should have a cache breaker here.
Pro: Make sure we definitely build our driver from a fresh repo
Con: Since we do not do this very often, not sure if this is necessary